src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -34,12 +34,17 @@
 
 class DoorLockServer
 {
+public:
+    // Where should this actually live?
+    char mPin[5];
```

Comment:
In the credential database provided by the app, presumably, with a defined API for the lock to talk to it...

Reply:
Made this comment a TODO saying this should be implemented when SetCredential is implemented.